corehq/apps/locations/tests/test_location_queries.py

```diff
@@ -149,13 +149,12 @@ def test_location_restricted_but_unassigned(self):
     def test_filter_by_user_input(self):
         self.restrict_user_to_assigned_locations(self.web_user)
 
+        mass_locs = SQLLocation.objects.filter_by_user_input(self.domain, "Massachusetts/")
+        self.assertIn(f'"domain" = {self.domain})', str(mass_locs.query))
```

Comment:
It took me a second to realize what exactly this was doing. I imagine a more "traditional" test that creates two locations in different domains with the same name, and asserts that only the location from the passed in domain is returned would be more intuitive? Just a personal preference though.

Reply:
Good point. https://github.com/dimagi/commcare-hq/pull/36216/commits/b938ff4d0c0de53668edc7368ef6593f8b96491a